canonical-web-and-design/www.ubuntu.com
# UA-I - update navigation

## Done

- Updated secondary nav per the read me
- Added redirects for moved plans-and-pricing, but leaving that in the nav per conversation with Thibuat (add pricing in /support secondary) for pricing.
- Updated redirects to old /support/plans-and-pricing
- Updated links in pages to new /pricing section
- Updates some links on the Enterprise drop-down navigation

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- [copy doc of /pricing section](https://docs.google.com/document/d/1llg7yZOzPMiCdCq2XgMvwYyTEqAo1y8t0svbX-0Eb4g/edit), [copy doc of enterprise dropdown](https://docs.google.com/document/d/1YBdQvLuqEpEQr_QqyycxMhZr4OaLapMlNJyYKCmPJsQ/edit#)


## Issue / Card

Fixes #5014
